pkg/meshsync/cluster/informers/depoyment.go

```diff
@@ -14,26 +15,32 @@ func (c *Cluster) DeploymentInformer() cache.SharedIndexInformer {
 	// register event handlers
 	deploymentInformer.AddEventHandler(
 		cache.ResourceEventHandlerFuncs{
-			AddFunc:    addDeployment,
-			UpdateFunc: updateDeployment,
-			DeleteFunc: deleteDeployment,
+			AddFunc: func(obj interface{}) {
+				deployment := obj.(*v1.Deployment)
+				log.Printf("Deployment Named: %s - added", deployment.Name)
```

Comment:
Is this the logger from MeshKit? What is that logger?

Reply:
No, it will be covered in different PR. 
It is the logger from the Go standard library.